src/measure_theory/lp_space.lean

```diff
@@ -97,6 +97,45 @@ end zero
 @[simp] lemma snorm_neg {f : α → F} : snorm (-f) p μ = snorm f p μ :=
 by simp [snorm]
 
+section opens_measurable_space
+variable [opens_measurable_space E]
+
+lemma snorm_zero_ae {f : α → E} (hp0_lt : 0 < p) (hf : measurable f) (hf_zero : f =ᵐ[μ] 0) :
```

Comment:
Wouldn't `p ≠ 0` be sufficient for this one, and no assumption on `p` be sufficient for the next one?

Reply:
I think that you are right for this one, but the next one needs `0 <= p`, or we could could have snorm equal to zero because the integral is infinite and `p < 0`.